vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -1654,7 +1654,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
             if (null != mRoom) {
                 RoomMember member = mRoom.getMember(mSession.getMyUserId());
                 // kicked / banned room
-                if ((null != member) && member.kickedOrBanned()) {
+                if ((null != member) && member.kickedOrBanned() && null != menu.findItem(R.id.ic_action_room_leave)) {
```

Comment:
This check is necessary because the menu is in fact not inflated (see the line `//getMenuInflater().inflate(R.menu.vector_room, menu);` above).
So yes, this fixes the crash, but maybe a better solution is to comment out all this block.